### PR TITLE
Dashboard Importer - Make trailing slash and project in import.sh optional

### PR DESCRIPTION
**Pull request by stevezease**

### CODE REVIEW

File: scripts/dashboard-importer/import.sh [hunk 1]
```diff
@@ -14,36 +14,34 @@
...
   echo "Error: No JSON_PATH included"
-  echo "Syntax: import.sh <JSON_PATH> <PROJECT>"
-  exit
-fi
-
-if [ "$PROJECT" = "" ]
-then
-  echo "Error: No PROJECT included"
-  echo "Syntax: import.sh <JSON_PATH> <PROJECT>"
+  echo "Syntax: ./import.sh <JSON_PATH> <PROJECT>"
```
Comment (yqlu) [inline scripts/dashboard-importer/import.sh:23]: Can this echo reflect that PROJECT is optional?

File: scripts/dashboard-importer/import.sh [hunk 2]
```diff
@@ -14,36 +14,34 @@
...
 CONVERSION_OUTPUT=$(node dist/convert.js dashboards $JSON_PATH)
 echo "$CONVERSION_OUTPUT"
-echo
-echo "Conversion Complete. Proceeding to uploading..."
 
-# Extract output directory from console output
-DIRECTORY=$(grep -Po '(reports.*/)' <<< "$CONVERSION_OUTPUT" | sed -n '2p')
-echo
-echo -e "Now running: \033[34m./upload.sh $DIRECTORY $PROJECT\033[0m\n"
+# If project provided then upload the converted daashboard to the project
```
Comment (yqlu) [inline scripts/dashboard-importer/import.sh:31]: dashboard

File: scripts/dashboard-importer/import.sh [hunk 3]
```diff
@@ -14,36 +14,34 @@
...
+if [ -n "$2" ]
+then
+  echo
+  echo "Conversion Complete. Proceeding to uploading..."
```
Comment (yqlu) [inline scripts/dashboard-importer/import.sh:35]: Let's be consistent with title case vs sentence case. Also, "Proceeding to upload..."

**Review by yqlu — COMMENTED** [review]
Just some nitpicks

File: scripts/dashboard-importer/import.sh (hunk 1 above)
Comment (stevezease) [reply scripts/dashboard-importer/import.sh]: Added  OPTIONAL_ in front of PROJECT, not sure if there's a better way, but open to suggestions!

File: scripts/dashboard-importer/import.sh (hunk 2 above)
Comment (stevezease) [reply scripts/dashboard-importer/import.sh]: Fixed!

**Review by yqlu — APPROVED** [review]

**Pull request merged**